Packs/DemistoRESTAPI/Integrations/CoreRESTAPI/CoreRESTAPI_description.md

```diff
@@ -1 +1,3 @@
-Creating API keys is done in the Cortex XSIAM interface, under Settings -> Configurations -> API Keys
\ No newline at end of file
+Creating API keys is done in the Cortex XSIAM interface, under Settings -> Configurations -> API Keys
```

Comment:
You can use <~XSOAR>
And <~XSIAM> 
to differentiate between XSOAR and XSIAM documentation